bin/kano-refresh-token

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# kano-refresh-token
+#
+# Python3 script to refresh the Kano World login token, using the new API.
+#
+
+import mercury
+import os
+import sys
+
+
+def test_refresh():
+    # Not implemented yet
+    pass
+
+
+if __name__ == '__main__':
+
+    rc = 1
+
+    # TODO: add docopt, move out code into a function, implement a test option
+    print('accessing mercury KanoWorld')
+    w = mercury.KanoWorld()
+    logged_in = w.is_logged_in(True)
+    if logged_in:
+        rc = 0
+        print('user is logged in, no action required')
```

Comment:
I don't understand... Do we not refresh the token while you're logged in? That's the main thing that keeps you logged in. Here don't you need to check if there exists a token instead?

Reply:
That's actually true. I am changing the logic here.